Content.Shared/Buckle/SharedBuckleSystem.Buckle.cs

```diff
@@ -436,9 +436,11 @@ public bool TryUnbuckle(EntityUid buckleUid, EntityUid userUid, bool force = fal
                 return false;
 
             // If the strap is a vehicle and the rider is not the person unbuckling, return.
-            if (TryComp<VehicleComponent>(strapUid, out var vehicle) &&
-                vehicle.Rider != userUid)
-                return false;
+            if (TryComp<VehicleComponent>(strapUid, out var vehicle) && vehicle.Rider != userUid)
+                if (_mobState.IsIncapacitated(buckleUid))
+                    TryUnbuckle(buckleUid, strapUid, true, buckleComp);
+                else
+                    return false;
```

Comment:
instead of this goofy recursive call, just only return false if the mob isn't incapacitated (make sure that the logic still works if the mob doesn't have mobstate component)